src/sdk/cart/useBuyButton.ts

```diff
@@ -16,10 +38,37 @@ export const useBuyButton = (item: Omit<CartItem, 'id'> | null) => {
         return
       }
 
+      sendAnalyticsEvent<VTEXAddToCartEvent>({
+        type: 'add_to_cart',
+        data: {
+          currency: code,
+          value: item.price * item.quantity,
```

Comment:
I think it's worth leaving a `TODO` comment here saying that in the future we may explore more robust ways of calculating the value. Maybe there is some gift item or some discount depending on the quantity. I don't think we should worry about that now, but it's valid to document the possibility.

Reply:
Fair enough. Done at https://github.com/vtex-sites/base.store/pull/53/commits/d46f50c9cdb1aeee730a2742744a2f8742c89b98